### PR TITLE
Improve Compliance with Parallel Model

### PR DESCRIPTION
**Pull request by fthaler**
Limit fusion of k-parallel execution with k-offsets.

### CODE REVIEW

**Comment by fthaler** [comment]
launch jenkins

**Comment by fthaler** [comment]
launch jenkins

File: include/gridtools/stencil/cpu_ifirst/entry_point.hpp [hunk 1]
```diff
@@ -31,6 +31,9 @@
 namespace gridtools {
     namespace stencil {
         namespace cpu_ifirst_backend {
+            template <class PlhInfo>
+            using get_extent_f = typename PlhInfo::extent_t;
```
Comment (anstaf) [inline include/gridtools/stencil/cpu_ifirst/entry_point.hpp:35]: You do not need to define this alias here. This is already at your service as `be_api::get_extent`

**Comment by fthaler** [comment]
launch jenkins

**Review by anstaf — APPROVED** [review]

**Pull request merged**